app.css

```diff
@@ -0,0 +1,56 @@
+* {
+  direction: rtl;
+  padding:0px;
+  margin:0px;
+}
```

Comment:
Hi,
2. you want me to do PR for every small change i do? how incremental it should be? give me an example
3. you mean that the logo shouldn't be part of the navbar?
4. you want me to put the "Learne" in h1?  

i didn't understand this comment, do you suggest to put the rtl rule in the body instead of *?